### PR TITLE
use pack instead algo

### PR DESCRIPTION
**Pull request by folliked**
Autobahn is OK . No regression. But unit tests not.

### CODE REVIEW

File: src/Utils/BitManipulation.php [hunk 1]
```diff
@@ -224,33 +224,20 @@ public static function bytesFromToString(string $frame, int $from, int $to, int
      * @param int|null $size  In bytes. This value should always be precise. Be careful if you don't !
      * @return string
      */
-    public static function intToString(int $frame, int $size = null) : string
+    public static function intToBinary(int $frame, int $size = 0) : string
```
Comment (Nek-) [inline src/Utils/BitManipulation.php]: `binaryString` if you want

File: src/Utils/BitManipulation.php [hunk 2]
```diff
@@ -224,33 +224,20 @@ public static function bytesFromToString(string $frame, int $from, int $to, int
...
-            }
+        $format = '*';
+
+        switch(true) {
+            case $size <= 2:
+                $format = 'n*';break;
+            case $size <= 4:
+                $format = 'N*';break;
+            case $size > 4:
+                $format = 'J*';break;
```
Comment (Nek-) [inline src/Utils/BitManipulation.php]: `break` on new line

File: src/Utils/BitManipulation.php (hunk 1 above)
Comment (Nek-) [inline src/Utils/BitManipulation.php]: The `$size = null` was not meaning 0.

Comment (folliked) [inline src/Utils/BitManipulation.php]: the switch does not process null value.

**Comment by folliked** [comment]
need what for merging ?

**Comment by Nek-** [comment]
It needs the tests pass !

**Comment by folliked** [comment]
They passed Oo, they are green , what else ?

**Comment by Nek-** [comment]
Duplicates #114

**Pull request closed without merging**